cinch/group_vars/cent7

```diff
@@ -9,10 +9,6 @@ _repositories:
 _download_repositories:
   - https://fedorapeople.org/~semyers/jenkins-rpm/jenkins1651.repo
 
-rpm_key_imports:
```

Comment:
This change is related to a separate issue.  We should either update the commit log and link to that issue or break this out into a separate PR.